contracts/libraries/JBFundingCycleMetadataResolver.sol

```diff
@@ -6,13 +6,19 @@ import './../interfaces/IJBFundingCycleDataSource.sol';
 import './../structs/JBFundingCycleMetadata.sol';
 
 library JBFundingCycleMetadataResolver {
+
+  /** 
+    @notice
+    Maximum value for token redemption, ballotRedemption rate.
+  */
+  uint private constant MAX_TOKEN_RATE = 10000;
```

Comment:
Please declare this in a libraries/JBConstants.sol file since it is used across files.